[2023-08-30T07:40:47.434+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_sale_data.create_result_table manual__2023-08-30T07:40:41.616474+00:00 [queued]>
[2023-08-30T07:40:47.443+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_sale_data.create_result_table manual__2023-08-30T07:40:41.616474+00:00 [queued]>
[2023-08-30T07:40:47.444+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-08-30T07:40:47.456+0000] {taskinstance.py:1327} INFO - Executing <Task(PostgresOperator): create_result_table> on 2023-08-30 07:40:41.616474+00:00
[2023-08-30T07:40:47.461+0000] {standard_task_runner.py:57} INFO - Started process 2537 to run task
[2023-08-30T07:40:47.463+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'process_sale_data', 'create_result_table', 'manual__2023-08-30T07:40:41.616474+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/dags.py', '--cfg-path', '/tmp/tmpaem2ztpg']
[2023-08-30T07:40:47.465+0000] {standard_task_runner.py:85} INFO - Job 53: Subtask create_result_table
[2023-08-30T07:40:47.518+0000] {task_command.py:410} INFO - Running <TaskInstance: process_sale_data.create_result_table manual__2023-08-30T07:40:41.616474+00:00 [running]> on host 888526fa4a36
[2023-08-30T07:40:47.603+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='process_sale_data' AIRFLOW_CTX_TASK_ID='create_result_table' AIRFLOW_CTX_EXECUTION_DATE='2023-08-30T07:40:41.616474+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-30T07:40:41.616474+00:00'
[2023-08-30T07:40:47.605+0000] {sql.py:265} INFO - Executing: CREATE TABLE IF NOT EXISTS consumption_alcoholic_20230830(
                category VARCHAR NOT NULL,
                sub_category  VARCHAR NOT NULL,
                aggregation_date  DATE NOT NULL,
                millions_of_dollar INT NOT NULL,
                pipeline_exc_datetime DATE NOT NULL
            );
            CREATE TABLE IF NOT EXISTS consumption_cereals_bakery_20230830(
                category VARCHAR NOT NULL,
                sub_category  VARCHAR NOT NULL,
                aggregation_date  DATE NOT NULL,
                millions_of_dollar INT NOT NULL,
                pipeline_exc_datetime DATE NOT NULL
            );
            CREATE TABLE IF NOT EXISTS consumption_meats_poultry_20230830(
                category VARCHAR NOT NULL,
                sub_category  VARCHAR NOT NULL,
                aggregation_date  DATE NOT NULL,
                millions_of_dollar INT NOT NULL,
                pipeline_exc_datetime DATE NOT NULL
            );      
            COPY consumption_alcoholic_20230830
                FROM 'consumption_alcoholic_20230830.csv'
                DELIMITER ','
                CSV HEADER;

            COPY consumption_cereals_bakery_20230830
                FROM 'consumption_cereals_bakery_20230830.csv'
                DELIMITER ','
                CSV HEADER;
            COPY consumption_meats_poultry_20230830
                FROM 'consumption_meats_poultry_20230830.csv'
                DELIMITER ','
                CSV HEADER;
[2023-08-30T07:40:47.613+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-08-30T07:40:47.679+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-08-30T07:40:47.684+0000] {sql.py:375} INFO - Running statement: CREATE TABLE IF NOT EXISTS consumption_alcoholic_20230830(
                category VARCHAR NOT NULL,
                sub_category  VARCHAR NOT NULL,
                aggregation_date  DATE NOT NULL,
                millions_of_dollar INT NOT NULL,
                pipeline_exc_datetime DATE NOT NULL
            );
            CREATE TABLE IF NOT EXISTS consumption_cereals_bakery_20230830(
                category VARCHAR NOT NULL,
                sub_category  VARCHAR NOT NULL,
                aggregation_date  DATE NOT NULL,
                millions_of_dollar INT NOT NULL,
                pipeline_exc_datetime DATE NOT NULL
            );
            CREATE TABLE IF NOT EXISTS consumption_meats_poultry_20230830(
                category VARCHAR NOT NULL,
                sub_category  VARCHAR NOT NULL,
                aggregation_date  DATE NOT NULL,
                millions_of_dollar INT NOT NULL,
                pipeline_exc_datetime DATE NOT NULL
            );      
            COPY consumption_alcoholic_20230830
                FROM 'consumption_alcoholic_20230830.csv'
                DELIMITER ','
                CSV HEADER;

            COPY consumption_cereals_bakery_20230830
                FROM 'consumption_cereals_bakery_20230830.csv'
                DELIMITER ','
                CSV HEADER;
            COPY consumption_meats_poultry_20230830
                FROM 'consumption_meats_poultry_20230830.csv'
                DELIMITER ','
                CSV HEADER;, parameters: None
[2023-08-30T07:40:47.715+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 277, in execute
    **extra_kwargs,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 349, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 380, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UndefinedFile: could not open file "consumption_alcoholic_20230830.csv" for reading: No such file or directory
HINT:  COPY FROM instructs the PostgreSQL server process to read a file. You may want a client-side facility such as psql's \copy.

[2023-08-30T07:40:47.727+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=process_sale_data, task_id=create_result_table, execution_date=20230830T074041, start_date=20230830T074047, end_date=20230830T074047
[2023-08-30T07:40:47.944+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 53 for task create_result_table (could not open file "consumption_alcoholic_20230830.csv" for reading: No such file or directory
HINT:  COPY FROM instructs the PostgreSQL server process to read a file. You may want a client-side facility such as psql's \copy.
; 2537)
[2023-08-30T07:40:47.958+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-08-30T07:40:47.980+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
